src/accelerate/test_utils/scripts/external_deps/test_zero3_integration.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2021 The HuggingFace Team. All rights reserved.
```

Comment:
This is a new file, please use the right year:

```suggestion
# Copyright 2024 The HuggingFace Team. All rights reserved.
```

Reply:
updated